apollo-plugin/apollo-log4j2/README.md

```diff
@@ -0,0 +1,38 @@
+# Apollo Log4j2
+
+This module could let you integrate log4j2 with apollo easily.
+You could create a namespace `log42.xml`, then log4j2 will load the configuration from it.
```

Comment:
Maintain the document in https://github.com/apolloconfig/apollo/tree/master/docs is better?

Reply:
Sure, the docs will be updated if this PR was approved.